sdk/keyvault/keyvault-admin/CHANGELOG.md

```diff
@@ -1,15 +1,25 @@
 # Release History
 
-## 4.5.0-beta.2 (Unreleased)
+## 4.5.0 (Unreleased)
 
 ### Features Added
 
+Since 4.4.0:
+- Managed Identity can now be used in place of a SAS token to access the blob storage resource when performing backup and restore operations.
+
 ### Breaking Changes
 
+Since 4.5.0-beta.1:
+- Change signature of backup and restore operations to use an overload when using Managed Identity to access the blob storage resource. This means
+  `undefined` no longer has to be passed in the `sasToken` parameter in order to set additional request options when using Managed Identity.
+  This change is only breaking for customers using 4.5.0-beta.1 and does not impact customers using the previous GA version, 4.4.0.
+
```

Comment:
Be sure to run the `Prepare-Release.ps1` common script to not only remove these empty entries, but update the central database for the impending release.